lang/en.js

```diff
@@ -74,7 +74,7 @@
     "container.applydefaultsettingscard.button": "Apply",
     "container.applydefaultsettingscard.success": "Your default settings have been successfully set.",
     "container.appNavigation.home": "Home",
-    "container.appNavigation.moresettings": "More Settings",
+    "container.appNavigation.moresettings": "Settings",
```

Comment:
I need to change all MoreSettings to settings if the name change will be accepted